projects/sel4/porting.md

```diff
@@ -7,79 +7,159 @@ SPDX-FileCopyrightText: 2020 seL4 Project a Series of LF Projects, LLC.
 
 # Porting seL4 to a new platform
 
+This section covers porting a new ARM or RISC-V platform to seL4. You can check the list of supported seL4 platforms [here](https://docs.sel4.systems/Hardware/).
```

Comment:
Also: it might be a good idea to have a link back to the general instructions (https://docs.sel4.systems/projects/sel4/kernel-contribution.html), which mention RFC, communication etc and/or the guidelines (https://docs.sel4.systems/processes/contributing.html)

Reply:
Yes, good point. I added those to the bottom of the page under the contributing section.